Script/AtomicEditor/editor/Preferences.ts

```diff
@@ -351,13 +402,19 @@ class PreferencesFormat {
             updatedMissingDefaults = true;
         }
 
+        if (!prefs.uiData) {
+            prefs.uiData = this.uiData;
+            updatedMissingDefaults = true;
+        }
+
```

Comment:
In here, you can add any missing defaults. ie
```
} else {
  prefs.uiData.skinPath = prefs.uiData.skinPath || this.uiData.skinPath;
  ...
}
```
The syntax of ``` value = valueA || valueB;``` means, use the value from valueA, unless it's ```falsy``` or ```null```, then use the value from valueB.  So what this does is will use whatever value is currently in the file unless it's missing, then it will use the value in the defaults.